internal/testbed/integration/filtering_test.go

```diff
@@ -155,6 +155,163 @@ func TestFilteringCreditCard(t *testing.T) {
 	}
 }
 
+func TestFilteringLogBody(t *testing.T) {
+	logsCCPlain := []string{
+		"card_master_spaces1 2367 8901 2345 6789",
```

Comment:
Maybe we can add test cases for `<text> <number> <text>` or just `<number> <text>` (see my other comment)

Reply:
done